[MTM-64335] Update working with dashboard docs to include all inventory roles able to edit dashboard (#3581)
SoftwareAG/c8y-docs: content/cockpit/working-with-dashboards.md

4. In the **Availability** section, specify which users have access to the dashboard based on global roles. By default, all available global roles are selected, which means that a user with at least one such role has access to the dashboard.

   {{< c8y-admon-info >}}

> - Dashboard are always visible to its owner and to users with ADMIN permission for the permission types "Inventory" or "Managed object".

Reviewer: ```suggestion
- Dashboards are always visible to their owner and to users with ADMIN permission for the permission types "Inventory" or "Managed object".
```

> - Dashboard are always visible to its owner and to users with ADMIN permission for the permission types "Inventory" and "Managed object".

Reviewer 2: I guess having the `ROLE_INVENTORY_ADMIN` or `ROLE_MANAGED_OBJECT_ADMIN` is sufficient, you do not need to have both of them, right?
```suggestion
- Dashboard are always visible to its owner and to users with ADMIN permission for the permission types "Inventory" or "Managed object".
```

Author: Hmm it actually you're right, it may be confusing even without saying: `for both the 'Inventory' and 'Managed object' permission types."`

Basically it checks for `any` role from these two, so in the end it could be `and/or`, but I'm not sure which one sounds better:
```suggestion
- Dashboard are always visible to its owner and to users with ADMIN permission for the permission types "Inventory" and/or "Managed object".
```
or
```suggestion
- Dashboard are always visible to its owner and to users with ADMIN permission for the permission types "Inventory" or "Managed object" (or both).
```
Otherwise, imho, it may sound like user needs to pick.. or I am overthinking it :)
